### PR TITLE
fix(transitive): provides transitive support for maven

### PR DESCRIPTION
**Pull request by invincibleJai**
- provides transitive support for maven ecosystem
- refactored stack analyses flow for a file and workspace
- removes license payload and extra findFiles
- includes path module

### CODE REVIEW

File: src/ProjectDataProvider.ts [hunk 1]
```diff
@@ -17,38 +17,18 @@ export module ProjectDataProvider {
       } else {
         vscodeRootpath += '/';
       }
+      const filepath = vscodeRootpath + `target/dependencies.txt`;
```
Comment (sudhirverma) [inline src/ProjectDataProvider.ts]: is this filepath works on windows?

**Comment by sudhirverma** [comment]
It will be good if you have build check for windows also

**Comment by invincibleJai** [comment]
@sudhirverma thanks, have done the changes pls verify

File: package.json [hunk 2]
```diff
@@ -188,7 +188,7 @@
   "dependencies": {
     "fabric8-analytics-lsp-server": "0.1.48",
     "fs": "0.0.1-security",
-    "path": "^0.12.7",
+    "path": "0.12.7",
```
Comment (sudhirverma) [inline package.json]: don't use npm package `path` nodejs provide that option

Comment (invincibleJai) [reply package.json]: yes am aware of that, used it as I took https://github.com/redhat-developer/vscode-adapters/blob/master/package.json#L244 as reference ;)

**Pull request merged**